content/uc-doc/api_sdk/rest_api/quickstart.md

```diff
@@ -82,20 +82,20 @@ To that end:
 
 Let's ask wazo-auth for an authentication token:
 
-1. Choose the `wazo-auth` service in the list of REST APIs
-2. In the top-right text box of the page (left to the "Explore" button), fill "token" with the
-   `rest-api-test:password`: those credentials are the ones from the Web Services Access you created
-   earlier.
-3. Go to the `POST /tokens` section and click on the yellow box to the right of the `body`
+1. Open `wazo-auth` spec in our [API Console](/documentation/console/authentication)
+2. In the first field, please enter your stack hostname (ex: `https://mystack.example`)
+3. In the last field, please enter `username:password` : those credentials are the ones from
+   the Web Services Access you created earlier.
+4. Go to the `POST /tokens` section and click on the yellow box to the right of the `body`
```

Comment:
To accept this answer we need to change the previous section that explain how to use the embedded swagger-ui  and add notice that no information will go through our server
The downside with the wazo-platform.org/api is that is not versioned compared to the embedded one